src/main/java/org/fundacionjala/sfdc/pageobjects/opportunities/SFONewModifyPage.java

```diff
@@ -0,0 +1,81 @@
+package org.fundacionjala.sfdc.pageobjects.opportunities;
+
+import org.fundacionjala.sfdc.pageobjects.SFNewModify;
+import org.fundacionjala.sfdc.util.CommonActions;
+import org.fundacionjala.sfdc.util.Value;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * SFONewModifyPage.
+ */
+public class SFONewModifyPage extends SFNewModify {
+
+    @FindBy(how = How.CSS, using = "#opp3")
+    private WebElement opportunityName;
+
+    @FindBy(how = How.CSS, using = "#opp9")
+    private WebElement opportunityCloseDate;
+
+    @FindBy(how = How.CSS, using = "#opp11")
+    private WebElement opportunityStage;
+
+
+    /**
+     * @param opportunityName setOpportunityName.
+     */
+    public void setOpportunityName(final String opportunityName) {
+        this.opportunityName.clear();
+        this.opportunityName.sendKeys(opportunityName);
+
+    }
+
+    /**
+     * @param closeDate setOpportunityCloseDate.
+     */
+    public void setOpportunityCloseDate(final String closeDate) {
+        this.opportunityCloseDate.clear();
+        CommonActions.getElement(opportunityCloseDate);
+        this.opportunityCloseDate.sendKeys(closeDate);
+    }
+
+    /**
+     * @param stage setOpportunityStage.
+     */
+    public void setOpportunityStage(final String stage) {
+        //CommonActions.clickElement(opportunityStage);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)